test/test_validation.py

```diff
@@ -62,6 +63,17 @@ def test_as_filename_wrong_type(self):
         with self.assertRaises(confuse.ConfigTypeError):
             config['foo'].as_filename()
 
+    def test_as_path(self):
+        config = _root({'foo': 'foo/bar'})
+        path = os.path.join(os.getcwd(), 'foo', 'bar')
+        if sys.version_info[0] != 3:
+            with self.assertRaises(confuse.ConfigTypeError):
+                value = config['foo'].as_path()
+        else:
+            value = config['foo'].as_path()
+            path = pathlib.Path(path)
```

Comment:
Looks like this needs an `import pathlib`! And perhaps we should add some machinery to skip this test if pathlib is not available.

Reply:
I mixed some commits up, sorry, repairing...

Reply:
Now I use the `try / except ImportError` in the test as well.